src/curveToken.ts

```diff
@@ -0,0 +1,153 @@
+import axios from "axios";
+import hre from "hardhat";
+import { TokenTag } from "src/tags";
+import { CurveLpToken } from "./types";
+import { ethers } from "ethers";
+
+export const provider = hre.ethers.provider;
+const etherscanProvider = new ethers.providers.EtherscanProvider(
+  1,
+  process.env.ETHERSCAN_API_KEY
+);
+
+async function getGauges(): Promise<
```

Comment:
comment on what a Gauge is.  I'm still not clear after reading through curve's GH repo haha